docs/source/instructions_lassen.rst

```diff
@@ -0,0 +1,116 @@
+Instructions for running DLIO Benchmark on Lassen, LLC.
+=============
+
+'''''''''''''''''''''''
+Installation
+'''''''''''''''''''''''
+On Lassen on the login node: 
+
+* **Clone the github repository**:
+
+.. code-block:: bash
+
+	git clone https://github.com/argonne-lcf/dlio_benchmark
+	cd dlio_benchmark/
+
+* **Use conda**:
+
+.. code-block:: bash
+
+	# Setup the required channels:
+	conda config --prepend channels https://public.dhe.ibm.com/ibmdl/export/pub/software/server/ibm-ai/conda/
+
+	# Create and activate environment
+	conda env create --prefix ./dlio_env_ppc --file environment-ppc.yaml --force
+	conda activate ./dlio_env_ppc
+
+	#Install other dependencies and make sure it finishes successfully with no errors:
+	python -m pip install .
+
+
+**NOTE**: if problem with mpi4py make sure that mpi is pointing to the right version of gcc.
+Do not install packages using the $conda install command but rather install all required versions of packages.
+To check versions of mpicc and gcc:
+
+.. code-block:: bash
+
+	gcc --version
+	mpicc --version
+
+To specify a new link for gcc:
+
+.. code-block:: bash
+
+	which mpicc
+	export CC='which mpicc'
+	export CXX=mpic++
+
+'''''''''''''''''''''''
+Generate synthetic data that DLIO will use
+'''''''''''''''''''''''
+
+**On Lassen generate data with the use of JSRUN scheduler**:
+* Arguments to use:
+1) --bind packed:4 (to bind tasks with 4 CPUs)
+2) --smpiargs="-gpu" (enables gpu support)
+3) --nrs x (allocation of x node, it can be set to to 1, 2, 4 etc On Lassen we have 756 compute nodes)
+4) --rs_per_host 1 (resources per node)
+5) --tasks_per_rs y (y processes per resourse set/per node, it can be set to to 1, 2, 4 etc On Lassen we have 44 processes per node)
+6) --launch_distribution packed (specify how tasks are started on the available resource sets within the allocation. Packed assigns task to the first resource set until each CPU in the resource set is assigned to a task, and then starts assigning tasks to the second resource set, third resource set, fourth resource set (and so on))
+7) --cpu_per_rs ALL_CPUS (each resource set contains the number of CPUs that are available on each compute node)
+8) --gpu_per_rs ALL_GPUS (each resource set contains the number of GPUs that are available on each compute node)
+
+For more information on these arguments please turn to: https://www.ibm.com/docs/en/spectrum-lsf/10.1.0?topic=SSWRJV_10.1.0/jsm/jsrun.htm
```

Comment:
Add a **Note** saying that Lassen machine has a custom wrapper over jsrun  which is also called ``jsrun`` which is used by default by the system.

Reply:
Fixed